examples/step-11/step-11.cc

```diff
@@ -161,11 +164,9 @@ namespace Step11
     // <code>std::find</code> returns an iterator to this element), and
     // computing its distance to the overall first element in the array to get
     // its index:
-    const unsigned int first_boundary_dof
-      = std::distance (boundary_dofs.begin(),
-                       std::find (boundary_dofs.begin(),
-                                  boundary_dofs.end(),
-                                  true));
+    const unsigned int first_boundary_dof = std::distance(
+      boundary_dofs.begin(),
+      std::find(boundary_dofs.begin(), boundary_dofs.end(), true));
```

Comment:
ugh

Reply:
I actually like this particular reformatting. :-) It is easier to read because the nested arguments are on one line.

Reply:
Oh...

Here, the function being called (`std::distance`) is horizontally printed *after* its arguments. I find this difficult to read.

I have no opinions about the arguments to `std::find`, but the arguments to `std::distance` are difficult for me.